intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -238,6 +238,36 @@ \subsection{When will my job start?}
 It might be beneficial to request less resources (e.g. not requesting all cores
 in a workernode), since the scheduler often finds a ``gap'' to fit the job into
 more easily.
+
+\fi
+
+\ifgent
+
+\subsection{Specifying the cluster on which to run}
+\label{subsec:specifying-the-cluster-on-which-to-run}
+
+To use other clusters, you can swap the \verb|cluster| modules.
```

Comment:
@redfast00 `modules` -> `module` (there can only be a single `cluster` module loaded at a time)

Maybe add an entry in the troubleshooting chapter on that (under `\ifgent`), with the error produced by `module load cluster/swalot` when `cluster/delcatty` is loaded?